frameworks/elastic/tests/test_tls.py

```diff
@@ -60,6 +61,29 @@ def elastic_service_tls(service_account):
     sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
 
 
+@pytest.fixture(scope='module')
+def kibana_application_tls(elastic_service_tls):
+    elasticsearch_url = "https://" + sdk_hosts.vip_host(config.SERVICE_NAME, "coordinator", 9200)
+
+    sdk_install.install(
+        config.KIBANA_PACKAGE_NAME,
+        service_name=config.KIBANA_PACKAGE_NAME,
+        expected_running_tasks=0,
+        additional_options={
+            "kibana": {
+                "xpack_enabled": True,
+                "elasticsearch_tls": True,
+                "elasticsearch_url": elasticsearch_url
+            }
+        },
+        timeout_seconds=config.DEFAULT_KIBANA_TIMEOUT,
+        wait_for_deployment=False)
+
+    yield
+
+    sdk_install.uninstall(config.KIBANA_PACKAGE_NAME, config.KIBANA_PACKAGE_NAME)
```

Comment:
We may want a `try - finally` block here to ensure that the uninstall is executed even if there is a failure.

Reply:
(and above for elastic!)

Reply:
https://github.com/mesosphere/dcos-commons/pull/2349/commits/e07b1a750211a6faa1a2e50579a4adb51cebad48